script.js

```diff
@@ -1,31 +1,26 @@
 var guesses = [];
 var misses = 0;
-var word = "ABALONE";
+var word = ['A','B','A','L','O','N','E'];
 
 function updateWord() {
   $("#word").empty();
-  for (var i = 0; i < word.length; i++) {
-    var letter = word.charAt(i);
+  word.forEach(function (letter) {
```

Comment:
Read: "**For each** letter in word, if guesses **includes** the letter, **append** the letter - otherwise, **append** a blank."